lab-ron/app/html/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html ng-app="demoApp">
+  <head>
+    <link rel="stylesheet" type="text/css" href="bundle.css">
+    <script src="bundle.js"></script>
+    <title>Lab24-25-Adventure Game</title>
+  </head>
+  <body>
+    <main ng-controller="gameController as gameCtrl">
+      <form novalidate name="gameForm">
+        <select ng-model="gameCtrl.gameForm.direction">
+          <option value="north"> North</option>
+          <option value="south"> South</option>
+          <option value="east"> East</option>
+          <option value="west"> West</option>
+        </select>
+        <input type="button" ng-click="gameCtrl.moveDirection(gameCtrl.gameForm.direction)" value="Move">
+        <input type="button" ng-disabled="!gameCtrl.room.monster" ng-click="gameCtrl.attackMonster()"value="Attack">
+      </form>
+      <section>
+        <h2>{{gameCtrl.player.name}}</h2>
+        <h3>HP:</h3><p>{{gameCtrl.player.hp}}</p>
```

Comment:
I really like that you're describing what you're showing! That's great UX. If you want to go further into that stuff, you should consider using more semantic html---the [label element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) exists for this purpose.